ggml.c

```diff
@@ -201,9 +201,11 @@ inline static void* ggml_aligned_malloc(size_t size) {
 #endif
 #elif defined(GGML_USE_OPENBLAS)
 #include <cblas.h>
-#elif defined(GGML_USE_CUBLAS) | defined(GGML_USE_HIPBLAS)
+#endif
+#if defined(GGML_USE_CUBLAS) || defined(GGML_USE_HIPBLAS)
```

Comment:
Could this be left with just the `GGML_USE_CUBLAS` part?

Reply:
yes it can I was about to do that and have pushed it now